site/index.html

```diff
@@ -77,6 +77,59 @@ <h3>Haiku</h3>
         <span class="author">– Sachin Tendulkar</span>
       </p>
     </section>
+
+    <!-- Table Section for Lab 4 -->
+    <div id="making-a-table">
+      <table>
+        <caption>Table 1. Current Courses</caption>
+        <thead>
+        <tr>
+          <th scope="col">Course Code</th>
+          <th scope="col">Course Title</th>
+          <th scope="col">Credit Hours</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+          <th scope="row">CSA103</th>
+          <td>Business Applications I</td>
+          <td>4</td>
+        </tr>
```

Comment:
> 5. `Lab 4 > Making a table > the table > body has > Fall Semester`

This also could've been set to Spring
```suggestion
        <tr>
          <th scope="rowgroup" rowspan="1">Fall</th>
          <th scope="row">CSA103</th>
          <td>Business Applications I</td>
          <td>4</td>
        </tr>
```